nbs/06_transform.ipynb

```diff
@@ -10,7 +10,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #79.        for lhs_src, lhs_target in rule.edges_to_remove(): 
</pre><p>same thing in this function. Do we really need all the different combinations of regular matches and collections as separate blocks of logic? this function is getting really hard to read.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/graph_rewrite/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>